.github/workflows/stryker.yml

```
@@ -0,0 +1,28 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: run Stryker - Node.js CI
+
+on:
+  push:
+    branches: [ master, stryker-* ]
```

I did make this more general to run Stryker based on `stryker-*` branches. This is probably only needed for testing, don't know if it may come in handy for something else in the future. I would be open to removing it.

```suggestion
    branches: [ master ]
```